src/main/scala/module/RhwRuleGenerator.scala

```diff
@@ -62,7 +62,9 @@ class RhwRuleGenerator(var context: RuleTransducer.Context) extends RuleGenerato
 
       // TODO filtering
       for (minor <- Network.values if minor != Subway && !isHrw(minor) && (main.isRhw || minor.isRhw ||
-           (main.isNwm && (minor.isRhw || minor.isNwm || minor.base.isEmpty))
+           (main.isNwm && (minor.isRhw || minor.isNwm || minor.base.isEmpty)) ||
+           (main.isNwm && isSingleTile(main) && main.height == 0 && (minor == L1Dtr || minor == L2Dtr)) ||
+           (main.isRhw && isSingleTile(main) && main.height == 0 && (minor == L1Dtr || minor == L2Dtr))
```

Comment:
This line (67) has no effect, since line 64 already contains `main.isRhw` by itself.

As for the previous line (66), why the restriction to _single-tile_ NWM networks?

Reply:
Good catch with line 67, seems completely unnecessary.  With line 66, I restricted the rules to single-tile NWMs because the intersections between L1Dtr/L2Dtr and multi-tile NWMs are not yet defined in any resolver.  They'll be added to the RRW resolver eventually.

Reply:
I also modified the rhwIntersectionAllowed function to do the same filtering for RHW networks for the same reason.

Reply:
The redundant line 67 has been removed.